modules/bridge/src/main/ts/ephox/bridge/components/dialog/Iframe.ts

```diff
@@ -6,15 +6,18 @@ import { FormComponentWithLabel, formComponentWithLabelFields, FormComponentWith
 export interface IframeSpec extends FormComponentWithLabelSpec {
   type: 'iframe';
   sandboxed?: boolean;
+  opaque?: boolean;
```

Comment:
This name is a little inconsistent, so I'd suggest we call this `transparent` instead. That's the CSS name and similar to the old IE `allowtransparency` iframe attribute name as well. That also means we'd need to set the default to `true` below.